scripts/data_collection/update_google_sheet.py

```diff
@@ -73,7 +73,7 @@ def upload_csv_file(worksheet, filename):
     logger.info(f"Uploading file {filename}")
 
     # Read CSV and update
-    with open(filename, "r") as file:
```

Comment:
Again? So should we be explicit or not?:)

Reply:
I guess it's by IDE wrongly configured. Fixing it

Reply:
fixed @onmete